### PR TITLE
Update to Nielsen usability heuristics: Initial commit

### PR DESCRIPTION
**Pull request by kchprog**
Partial update of Nielsen Heuristics page, including citations and additional context. Work in progress.

### CODE REVIEW

**Review by franciscojben — COMMENTED** [review]
It would be helpful to the reader to give an example of Nielsen's usability heuristics and how it satisfies the five components of "Learnability", "Efficiency", "Memorability", "Errors", and "Satisfaction"

**Review by franciscojben — COMMENTED** [review]
For section "1: Visibility of system status", the example of an application providing a loading screen would be beneficial to keep.

**Comment by franciscojben** [comment]
Overall, really good initial commit.  I love how citations are embedded in the markdown file.

**Review by franciscojben — CHANGES_REQUESTED** [review]
Finish the other sections.

File: Topics/User_Experience/Usability_Heuristics.md [hunk 1]
```diff
@@ -1,60 +1,236 @@
...
+
+Reducing Mistakes:
+
+When users interact with an application, they have a goal they want to achieve. Based on the user's understanding of how the system operates, the user forms a plan. Then they take action based on this plan, and see whether their goal has been achieved. Mistakes occur when there is a mismatch between user understanding of the software and how the software actually works, leading the user to create a plan that results in an unexpected outcome (10). Generally, to reduce mistakes it is necessary to both convey the designer's model of the application to the user and to design the application in such a way that it is intuitive and easy to learn.
+
+
+**Example:**
```
Comment (franciscojben) [inline Topics/User_Experience/Usability_Heuristics.md:129]: You gave some nice examples earlier so I don't think you need to include this line.

File: Topics/User_Experience/Usability_Heuristics.md [hunk 2]
```diff
@@ -66,3 +242,32 @@ Heuristics Applied to Everyday Life: https://www.zenhaiku.com/archives/usability
...
+
+10. Laubheimer, P. (2015, September 7). Preventing user errors: Avoiding conscious mistakes. Nielsen Norman Group. https://www.nngroup.com/articles/user-mistakes/ 
+
+11. Budiu, R. (2014, July 6). Memory recognition and recall in user interfaces. Nielsen Norman Group. https://www.nngroup.com/articles/recognition-and-recall/ 
+
+12. Laubheimer, P. (2022, November 22). Flexibility and efficiency of use (usability heuristic #7). Nielsen Norman Group. https://www.nngroup.com/articles/flexibility-efficiency-heuristic/ 
+
+13. Fessenden, T. (2021, January 24). Aesthetic and minimalist design (usability heuristic #8). Nielsen Norman Group. https://www.nngroup.com/articles/aesthetic-minimalist-design/ 
+
+14.
```
Comment (franciscojben) [inline Topics/User_Experience/Usability_Heuristics.md:273]: Not sure if there is a missing citation, could you please check if there were supposed to be a 14th entry here?

**Review by franciscojben — CHANGES_REQUESTED** [review]
Only some minor nitpicks.  Content of this page is well detailed.  LGTM.

File: Topics/User_Experience/Usability_Heuristics.md (hunk 1 above)
Comment (franciscojben) [reply Topics/User_Experience/Usability_Heuristics.md:129]: (The line I'm referring to is line 129 if it is unclear)

**Comment by kchprog** [comment]
> Finish the other sections.

I will be sure to complete the document, following the same pattern as before.

File: Topics/User_Experience/Usability_Heuristics.md (hunk 1 above)
Comment (kchprog) [reply Topics/User_Experience/Usability_Heuristics.md:129]: Thank you. I will remove this example, as it is not contributing anything novel to the document.

File: Topics/User_Experience/Usability_Heuristics.md (hunk 2 above)
Comment (kchprog) [reply Topics/User_Experience/Usability_Heuristics.md]: There is a 14th entry here. I have fixed this error. Good catch!

**Review by franciscojben — APPROVED** [review]
LGTM!  Feel free to merge.

**Pull request merged**